lib/measures/urban_geometry_creation/measure.rb

```diff
@@ -52,657 +53,32 @@ def description
   def modeler_description
     return ""
   end
-  
-  # define the arguments that the user will input
-  def arguments(model)
```

Comment:
@karinamzalez the arguments method is part of the measure and should stay with the measure